pkg/installmanager/testdata/pull-secret.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
i'm not 100% clear on how all this merged pull secret stuff works, so forgive me if i'm missing the big picture

the way i'm reading the purpose of this file is to represent the generated merged pull secret for test cases. if so, can we name this file 'merged-pull-secret.json'? and could we have it have two pull secret entries to make it clear that this represents the state post-merge?

Reply:
The installmanager does not have a concept of a merged pull secret. The installmanager just gets a pull secret file. There is no pre- and post-merge pull secret here.

Reply:
i now see that's the domain of the clusterdeployment controller